README.md

```diff
@@ -555,8 +554,72 @@ interface YoutubeVideo extends Node {
 }
 ```
 
+
 **YoutubeVideo** represents a video referenced by a Youtube URL.
 
+### `ClipSet`
+
+```ts
+interface ClipSet extends Node {
+	type: "clip-set"
+	id: string
+	autoplay: boolean
+	loop: boolean
+	muted: boolean
+	layoutWidth: 'in-line' | 'mid-grid' | 'full-grid'
```

Comment:
@adgad Consistently with what we have done with other PRs perhaps we can use `Extract` from the Layout here too